src/screens/OMPSwitch/ProfileSwitch.res

```diff
@@ -134,9 +134,10 @@ let make = () => {
     }
   }
 
-  let customPadding = "px-1 py-1"
+  let customPadding = ""
```

Comment:
can you remove this variable as it is not used anymore

Reply:
Yes removed the unused variable.